save-agent/src/linuxX64Main/kotlin/org/cqfn/save/agent/AgentConfiguration.kt

```diff
@@ -40,3 +40,17 @@ data class AgentConfiguration(
 data class HeartbeatConfig(
     val intervalMillis: Long,
 )
+
+/**
+ * Configuration for connection to backend service
+ *
+ * @property url URL of backend
+ * @property additionalDataEndpoint endpoint to post additional data (version etc.) to
+ * @property executionDataEndpoint endpoint to post execution data to
+ */
+@Serializable
+data class BackendConfig(
+    val url: String,
+    val additionalDataEndpoint: String = "saveAgentVersion",
```

Comment:
Why might endpoints in backend it be customizable?

Reply:
It's mostly not about customization, but rather about keeping the whole configuration in a single place, because we actually had an endpoint in backend, but in agent a different endpoint was hardcoded somewhere deep in logic.